Controller/CRUDController.php

```diff
@@ -270,11 +271,12 @@ public function editAction($id = null)
 
                 try {
                     $existingObject = $this->admin->update($submittedObject);
+                    $objectId = $this->admin->getNormalizedIdentifier($existingObject);
```

Comment:
When i see this code first time, i think that object id can change while update.
I fix this.